src/Text/Markdown/SlamDown/Halogen/Component.purs

```diff
@@ -413,19 +423,21 @@ renderFormElement config st id label field =
             SDIT.availableInputType config.browserFeatures tb
 
 secondsStep ∷ forall r i. HP.IProp r i
-secondsStep = refine $ HC.Attr M.Nothing (HC.attrName "step") "1"
+secondsStep = refine $ HC.attr (HC.AttrName "step") "1"
```

Comment:
It's in `dom-indexed`, but Halogen does not export `step` as a property. I've changed this though to use the properly propped property.